### PR TITLE
Schaake

### PR DESCRIPTION
**Pull request by elray1**
Some helper functions for dependence postprocessing.

### CODE REVIEW

File: tests/postpredict/test_apply_shuffle.py [hunk 1]
```diff
@@ -1,70 +1,37 @@
...
-# time dependence whereas that example is modeling spatial dependence.
 
-import numpy as np
 import polars as pl
-import pytest
 from polars.testing import assert_frame_equal
 from postpredict.dependence import TimeDependencePostprocessor
 
 
-@pytest.fixture
```
Comment (elray1) [inline tests/postpredict/test_apply_shuffle.py:18]: moved fixtures to conftest.py

File: src/postpredict/weighters.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+
+class EqualWeighter():
+    def __init__(self) -> None:
+        pass
+    
+    
+    def get_weights(X_train, X_test):
```
Comment (matthewcornell) [inline src/postpredict/weighters.py:9]: minor niggle: Python variable names conventionally start with a lower case letter

File: tests/postpredict/test_build_train_X_Y.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+from datetime import datetime, timedelta
+
+import polars as pl
+from polars.testing import assert_frame_equal
+from postpredict.dependence import TimeDependencePostprocessor
+
+
+def test_build_train_X_y_positive_horizons(obs_data, monkeypatch):
+    # we use monkeypatch to remove abstract methods from the
```
Comment (matthewcornell) [inline tests/postpredict/test_build_train_X_Y.py:11]: Seems a little tricky, but I haven't worked with ABCs in python.

**Review by matthewcornell — APPROVED** [review]
I can't really offer much in the way of comments b/c I don't understand the domain, nor am I familiar with polars. My best is a shaky LGTM. I'd be happy to sit in on an in-depth tour with @elray1 , if that would help.

**Pull request merged**